Controllers/ItemVMController.cs

```diff
@@ -0,0 +1,17 @@
+﻿using HowManyCaloriesIn.Models;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace HowManyCaloriesIn.Controllers
+{
+    public class ItemVMController : Controller
+    {
+        public IActionResult Index()
+        {
+            return View();
```

Comment:
You're returning an instance of the view without a model and then on the frontend, trying to access that model which is `null`.

You should be returning the viewmodel you're using on the frontend inside the view. Pseudocode example:

```
public IActionResult Index()
{
     var itemVMs = GetItemsFromDb(UserIdentity);
     return View(itemVMs);
}
```